packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -103,18 +103,25 @@ const useLazyLoadCategories = () => {
         }
     }
 }
-
+// This value represents the max age in milliseconds a customer can be before they are
+// no longer considered a "new" customer.
+// E.g. If a customers creation date is older than 2 seconds it will no longer be considered
+// a new customer.
+const NEW_CUSTOMER_MAX_AGE = 2 * 1000 // 2 seconds in milliseconds
```

Comment:
I'm concerned that we are measuring new customers in milliseconds. Feels error prone and fragile. Is there another way?

Reply:
afaik, there is no way to tell from data if a customer is new or recurring. @bendvc SInce you come up with this approach, maybe you can have some input

Reply:
Another approach would be to add the logic to the login calls, `login.mutate({onSuccess: () => {console.log('I know i'm not a new customer!')}})`

To do this, you'd probably need to DRY the login mutations since it's scattered in many components right now. 

(I'm not saying that this alternative is better, just wanna mention it.)

Reply:
@kevinxh I don't think that is accurate, when a user registers for an account, we immediately call login after that, so there is no way to tell if the login call is recurring or new

Reply:
@kevinxh @bfeister I looked into Kevin's suggestion, it is possible to trigger the logic when login is successful. We don't need to rely on the NEW_CUSTOMER_MAX_AGE.